myFunction().then().catch().finally(() => {
  // 성공/실패와 상관 없이 호출
})
